README.md

```diff
@@ -42,62 +41,38 @@ dependencies {
 
 ## Usage
 
-The following sections demonstrate how to retain instances in activities and fragments.
-
-For simplicity, all examples will retain the following class:
+The following sections demonstrate how to retain instances in activities and fragments. For simplicity, all examples will retain the following class:
```

Comment:
All my changes in this section have the main goal of decreasing the size so it's easier to go through it. This doesn't really help that much but I still thought it made sense. I think.